### PR TITLE
Emattson/finding my feet

### PR DESCRIPTION
**Pull request by emattson**
## Summary
Update setup so it works per instructions
Add .env.example
Update questions so we can add them with packs

### CODE REVIEW

File: server/lib/game/question_cache.ex [hunk 1]
```diff
@@ -33,15 +33,18 @@ defmodule Game.QuestionCache do
...
-          pack_name == pack
-        end) |> get_random_questions(10)
+    case Enum.empty?(questions) do 
+      true -> Database.Catalog.random_formatted_questions(10, pack)
+      false ->
+        case pack == "Variety" do
+          true -> questions |> get_random_questions(10)
+          false -> 
+            Enum.filter(questions, fn x ->
+                  pack_field = Enum.at(x, 2)
```
Comment (andyrewlee) [inline server/lib/game/question_cache.ex:43]: we should probably set up some sort of linting

shouldn't this block be indented back? from lines 43 and 46

File: Makefile [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	setup build test deploy console
 
 setup: install
-	cd server mix ecto.setup
+	cd server && mix ecto.setup
```
Comment (andyrewlee) [inline Makefile:9]: great catch!

**Comment by andyrewlee** [comment]
can we have a commit just fixing the `Makefile` and setup stuff?

we used to load questions from DB and then moved to Airtable which is why we have remnants of questions and seed data. we are going to move back into the DB from Airtable this weekend and will have to change the data model a lot. now there are different question types and answer types that we will have to model out.

**Comment by emattson** [comment]
Cool I'm going to close this and re-open a new PR with only the setup changes then

**Pull request closed without merging**